### ISSUE TITLE
FEEDBACK NEEDED FOR UPCOMING RELEASE

### ISSUE THREAD

**Issue by keeferrourke** [keeferrourke/la-capitaine-icon-theme#162]
### I'm finally gearing up for 0.5.0 release
La Capitaine has changed significantly since 0.4.0, so I'm looking for a little feedback to be included in the upcoming release notes.

I have only extensively tested these icons under the GNOME3 Shell and (less extensively) under XFCE with the Arc GTK and Clearlooks GTK themes. 

...So, what I'm wondering is, **are there any glaring issues to note before I make the next official release?** 

The release will come in the form of a signed tar.gz file, and will be distributed via the [Releases](https://github.com/keeferrourke/la-capitaine-icon-theme/releases) section of this Github repository, [my personal website](https://krourke.org/projects/art/la-capitaine-icon-theme), http://gnome-look.org (and other opendesktop subsidiaries), and on [Deviant Art](http://krourke.deviantart.com/art/La-Capitaine-624186412).

Please reply with any comments, and reference any issues that you think are important to close before the release so that I can assign them triage priorities.

Thanks again to everyone who has helped to support this project, to everyone who has contributed code, donated money, and lastly to the people that use this project so regularly. You guys inspire me to keep doing what I do :heart:

**Comment by linusg** [comment]
Hi Keefer,
Again, I want to thank you for this beautiful icon theme! I know it's not completely made on your own, but after all you're doing the most work and maintenance here. Good job!

I use Cinnamon, and have nothing to criticize. I see La Capitaine everywhere (almost), and when I don't see them I know it's not your fault. Hardcoders, f*** you! I have only two applications having **no** icon, Tor Browser and Fritzing (software for drawing electronics schematics). Dunno why, but I guess I can fix it by editing the `.desktop` files.

11 applications are remaining unthemed, and can be fixed by either symlinking, or aren't this important that they need a fresh icon *right* now. Guess what, LM standard apps included. :grin: 

I will create separate issues for these, guess you have enough to do with the high priorities right now.

Long story short: Awesome work, the important issues are already marked high priority, so I won't link any here. IMHO one icon more or less in 0.5.0 doesn't matter, hard-code tray support indeed does.

**Comment by keeferrourke** [comment]
@linusg Theming the Tor browser icon is difficult, because you *shouldn't* install that application from your distribution repositories. The Tor Project does some clever things to ensure anonymity and an always-up-to-date browser. I can likely provide instructions in Troubleshooting.md before I make the release &mdash; I know I had to play around with the "smart" desktop file a fair bit. I'll see what I can do to make it easier for people that don't want that ugly tor-browser xpm cluttering their desktops.

TO DO for upcoming release:
- [ ] seriously look into adding hard-code tray support

**Comment by linusg** [comment]
Fine, I didn't have installed it from the repos! Uncompressed and moved to home dir, done. If you need any help with the tray icons, let me know. I would be glad to help. (And currently, I have the time for it :wink: )

**Comment by cjdm** [comment]
First of all, thanks for this icon theme, I've been using it for a little while —since the release of Gnome 3.22 in Arch.
A couple of days ago I noticed that some symlinks were broken:
![one](https://user-images.githubusercontent.com/29413765/27510591-32f6bda4-58e2-11e7-846e-fd476e5dbdf2.png)
![two](https://user-images.githubusercontent.com/29413765/27510594-423e3ef4-58e2-11e7-9947-bbf2f62257a6.png)
I don't know if this is due to my installation from the AUR, so I'm not opening an issue. 
Regards.

**Comment by keeferrourke** [comment]
@cjdm I will take a look at this, thanks for pointing it out :)

**Comment by rzlx** [comment]
Hello keefe, thank you for this beautiful icon packs. I absolutely love it. But there's a problem. The battery, wifi, and volume indicator did not match with the panel. 
![2017-06-29-142115_1366x768_scrot](https://user-images.githubusercontent.com/29770791/27677258-295291fe-5cdb-11e7-96c5-d7272c3fe055.png)

I'm using Manjaro 17 LXDE. This is the only bug that I found. Other than that, everything is fine.
Can you fix it? 
Thanks in advance and sorry for my english.

**Comment by now-im** [comment]
Maybe this is me neat picking too much, there is some inconsistency in qt icons. They are not visible to common users but developers need them. I will be providing any information necessary. 
DE: KDE
![spectacle b19041](https://user-images.githubusercontent.com/22009014/27747342-84f2f14a-5dec-11e7-9f54-2c93ea12d5da.png)

**Comment by keeferrourke** [comment]
@now-im would you be able to provide the icon names for QT Interface Designer and D-bus Debugger? Just open a new issue following the format specified in the [wiki](https://github.com/keeferrourke/wiki).

**Comment by keeferrourke** [comment]
@rzlx if you pull the latest commits from github, and re-run the configure script, you should be prompted which theme variant to use depending on your panel theme. If you continue to have problems, please open a new issue.

**Comment by now-im** [comment]
Seems like qt designer is under [qt5-tools](https://www.archlinux.org/packages/extra/i686/qt5-tools/) and debugger under [qt5-base](https://www.archlinux.org/packages/extra/x86_64/qt5-base/files/).
do i post for them? otherwise i am not seeing any other ways.

**Comment by keeferrourke** [comment]
@now-im opening a new issue would be preferred.

**Comment by rzlx** [comment]
Thanks for your suggestion man. I have tried it, and it works. I'm sorry that I did not read the main page carefully :)

**Comment by now-im** [comment]
Open a new new issue then, but i would need your help because i don't know how can i gather the required info.

**Comment by keeferrourke** [comment]
@now-im please read the [Wiki](https://github.com/keeferrourke/la-capitaine-icon-theme/wiki) and [Troubleshooting](https://github.com/keeferrourke/la-capitaine-icon-theme/blob/master/Troubleshooting.md) to learn how to gather the required information.

**Comment by now-im** [comment]
Okay, i read the wiki not the troubleshooting methods. As far as I can see you have made icons for them then why are they not working?
![image](https://user-images.githubusercontent.com/22009014/27842338-15840a28-612b-11e7-977f-175ca7fec8e8.png)

**Comment by keeferrourke** [comment]
@now-im open a new issue and I will help you there. This is off-topic at this point.

**Comment by keeferrourke** [comment]
Closing discussion on this issue since I evidently have lots to do before 0.5.0. More changes to come after this release :)

**Comment by keeferrourke** [comment]
I'm going to freeze the code for v0.5.0 very soon. There have been lots of changes in this release, and I want to make it available ASAP. Hardcode-tray support is being moved to v0.6.0  release.
